src/modules/data/blocks/ticket/selectors.js

```diff
@@ -113,21 +113,18 @@ export const getTicketsCount = createSelector(
 	( allIds ) => allIds.length,
 );
 
-export const getTicketsIdsInBlocks = createSelector(
-	[ getTicketsArray ],
-	( tickets ) => tickets.reduce( ( accumulator, ticket ) => {
-		if ( ticket.ticketId !== 0 ) {
-			accumulator.push( ticket.ticketId );
-		}
-		return accumulator;
-	}, [] ),
-);
-
 export const hasTickets = createSelector(
 	[ getTicketsCount ],
 	( count ) => count > 0,
 );
 
+export const hasCreatedTickets = createSelector(
+	[ getTicketsArray ],
+	( tickets ) => tickets.reduce( ( hasCreated, ticket ) => (
```

Comment:
Are we trying to see if every item has been created, or just some of them? We can probably replace it with the `.every` or `.some` method if it's either of those cases

Reply:
We are trying to look if at least one is  `true`

Reply:
⬆️